host_reference/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+include_directories(../utils)
+
+# enable utils build a common library for all utils
+set(QUDA_REFERENCE_COMMON covdev_reference.cpp dslash_test_helpers.cpp hisq_force_reference.cpp 
+  dslash_reference.cpp wilson_dslash_reference.cpp clover_reference.cpp domain_wall_dslash_reference.cpp 
+  gauge_force_reference.cpp hisq_force_reference2.cpp staggered_dslash_reference.cpp)
+cuda_add_library(quda_reference STATIC ${QUDA_REFERENCE_COMMON})
+if(QUDA_QMP AND QUDA_DOWNLOAD_USQCD AND NOT QUDA_QMPHOME)
+  add_dependencies(quda_reference QMP)
+endif()
+
+if(QUDA_QDPJIT)
+  target_link_libraries(quda
```

Comment:
Curious, why is this here?

Reply:
This is because I copied the tests/CMakeLists.txt file and left it in. In the tests, I see that the building of tests is disabled when QDPJIT is enabled, so I don't know why it's the tests. Why is that bit of code needed in tests/CMakeLists.txt?